### PR TITLE
[CPU] Security fixes

### PR DESCRIPTION
**Pull request by v-Golubev**
### Tickets:
 - *69375*

### CODE REVIEW

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_reduce_node.cpp [hunk 1]
```diff
@@ -1825,7 +1825,8 @@ inline void MKLDNNReduceNode::reduce_kernel_process(const uint8_t *in_p, uint8_t
 }
 
 inline void MKLDNNReduceNode::reduce_kernel_post_process(uint8_t *out_ptr) {
-    const float divisor = static_cast<float>(IB * IC * ID * IH * IW / (OB * OC * OD * OH * OW));
+    const size_t divisor = IB * IC * ID * IH * IW / (OB * OC * OD * OH * OW);
+    const float casted_divisor = static_cast<float>(divisor);
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_reduce_node.cpp:1829]: casted_divisor looks embarrassing. I would prefer to update the name of the temporary variable, bot major one.

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_gather_elements_node.cpp [hunk 2]
```diff
@@ -69,6 +69,9 @@ MKLDNNGatherElementsNode::MKLDNNGatherElementsNode(const std::shared_ptr<ngraph:
             strideAx1Diff_ *= dataDims[i];
         strideAx1Diff_ -= strideAxDst_ * outputShape[axis_];
     }
+
+    Precision inDataPrecision = getOriginalInputPrecisionAtPort(dataIndex_);
```
[the remark this answers is not in the document]
Comment (v-Golubev) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_gather_elements_node.cpp]: Fixed, thank you!

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_reduce_node.cpp (hunk 1 above)
Comment (v-Golubev) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_reduce_node.cpp]: Temporary variable was renamed to `integerDivisor`, thanks

**Review by dmitry-gorokhov — APPROVED** [review]
Please rebase on latest master

**Pull request merged**